emu/src/memory/internal_memory.rs

```diff
@@ -135,6 +135,18 @@ impl InternalMemory {
         part_3 << 24_u32 | part_2 << 16_u32 | part_1 << 8_u32 | part_0
     }
 
+    pub fn write_word(&mut self, address: usize, value: u32) {
+        let part_0: u8 = (value & 0xFF).try_into().unwrap();
+        let part_1: u8 = ((value & (0xFF << 8)) >> 8).try_into().unwrap();
+        let part_2: u8 = ((value & (0xFF << 16)) >> 16).try_into().unwrap();
+        let part_3: u8 = ((value & (0xFF << 24)) >> 24).try_into().unwrap();
```

Comment:
For the sake of fun, can be better in terms of readability write something like
`value.get_bits(0..=3)`
`value.get_bits(4..=7)` ...

Reply:
I know but `get_bits` uses a for which I don't like in this case since it's slower. Maybe I can rewrite `get_bits` and then I can use it here

Reply:
Done